python/tests/conftest.py

```diff
@@ -828,9 +828,9 @@ def to_input_file(self):
 
     def create(self, overwrite: bool = False) -> OutputStream:
         output_file = open(self._path, "wb" if overwrite else "xb")
-        if not isinstance(output_file, OutputStream):
-            raise TypeError("Object returned from LocalOutputFile.create(...) does not match the OutputStream protocol.")
-        return output_file
+        # if not issubclass(type(output_file), OutputStream):
```

Comment:
Should we just remove these two commented lines?

Reply:
We actually need this line for mypy to assert that it is an outputstream. I now commented it out because it would never pass.